src/components/themeswitcher/ThemeSwitcher.vue

```diff
@@ -0,0 +1,47 @@
+<template>
```

Comment:
Could you add a short description what this component does?

Reply:
Yes sure. In the documentation or as a comment in the template?

Reply:
just as a short comment in the template

Reply:
Done!